remove settings from wazuh dashboard (#8762)
wazuh/wazuh-documentation: source/user-manual/wazuh-dashboard/settings.rst

The configuration file shows the default values for all of the possible options. You can edit this file, uncomment any of them, and apply the desired values. You can edit these settings from the Wazuh dashboard in **Dashboard management** > **App Settings** > **Configuration**.

.. note::

>    Starting with Wazuh 5.0.0, the dashboard no longer manages background jobs for monitoring or statistics. The related dashboard settings were removed and these tasks are now handled by Wazuh server and/or Wazuh indexer.
> 
>    Removed settings include: ``wazuh.monitoring.*``, ``cron.statistics.*``, and ``cron.prefix``.

Reviewer: ```suggestion
   Starting with Wazuh 5.0.0, the dashboard no longer manages background jobs for monitoring and statistics. The Wazuh server and indexer now handle these tasks. We removed the ``wazuh.monitoring.*``, ``cron.statistics.*``, and ``cron.prefix`` dashboard settings.
```

Reviewer: What about shortening this note a little further
